pipeline/tests/powdr_std.rs

```diff
@@ -388,13 +388,14 @@ mod reparse {
     /// but these tests panic if the field is too small. This is *probably*
     /// fine, because all of these tests have a similar variant that does
     /// run on Goldilocks.
-    const BLACKLIST: [&str; 6] = [
+    const BLACKLIST: [&str; 7] = [
         "std/bus_permutation_via_challenges.asm",
         "std/permutation_via_challenges.asm",
         "std/lookup_via_challenges.asm",
         "std/poseidon_bn254_test.asm",
         "std/split_bn254_test.asm",
         "std/bus_lookup_via_challenges.asm",
+        "std/fp4.asm",
```

Comment:
I don't think this is necessary!

Reply:
I got a reparsing error in the tests, and added this one

Reply:
What was the error?

Reply:
- I removed this one from the blacklist, and it seems it's not necessary
- I double-checked and it is a std::reparse error (something was wrong related to addition, i supposed it is about the extension field) but, not related to this blacklist.